sdk/keyvault/keyvault-common/package.json

```diff
@@ -31,7 +31,7 @@
     "integration-test:node": "echo skipped",
     "integration-test:node:no-timeout": "echo skipped",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "lint:fix": "eslint package.json src --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint:fix": "eslint src --ext .ts --fix --fix-type [problem,suggestion]",
     "lint": "eslint src --ext .ts -f html -o search-lintReport.html || exit 0",
```

Comment:
```suggestion
    "lint": "eslint src --ext .ts -f html -o keyvault-common-lintReport.html || exit 0",
```

Reply:
why is it that we do not want to lint the package.json file here?

Reply:
I will add it back, I just thought it is an internal package and our linting rules for package.json do not apply.